src/org/labkey/test/components/ui/entities/EntityBulkInsertDialog.java

```diff
@@ -381,10 +386,10 @@ public Input numericInput(String fieldKey)
             return new Input(inputEl, getDriver());
         }
 
-        public ReactDateTimePicker dateInput(String fieldKey)
+        public ReactDateTimePicker dateInput(String fieldName, String fieldKey)
```

Comment:
Annotations about nullability would be helpful here.

Reply:
yes, I'll add